sunpy/timeseries/timeseries_factory.py

```diff
@@ -521,7 +521,8 @@ class NoTimeSeriesFound(ValueError):
     `~sunpy.timeseries.TimeSeries` or files.
     """
 
-
 TimeSeries = TimeSeriesFactory(registry=GenericTimeSeries._registry,
                                default_widget_type=GenericTimeSeries,
                                additional_validation_functions=['is_datasource_for'])
+
+TimeSeriesFactory.__doc__ = TimeSeriesFactory.__doc__.format(list_packages = input_lines)
```

Comment:
I think this is ok. I would prefer if we can follow the style we use for the time types formatting.